README.md

```diff
@@ -8,6 +8,25 @@ Simple helpers for generating markdown with [Apep Javascript text generation lib
 $ npm install apep-md
 ```
 
+## Headers and Context
+    var pep = require("apep");
```

Comment:
The recommended approach is to use this module directly instead of extending apep. Change this to `var md = require('apep-md')`